bin/config-validator.rb

```diff
@@ -350,6 +350,25 @@ def check_rm_variables(manifest)
     STDOUT.puts "role manifest variable #{variable['name'].red} was not found in any role manifest template"
     @has_errors += 1
   end
+
+  manifest['configuration']['variables'].each do |variable|
```

Comment:
We should put any new validation in fissile instead of here (because that gets better tested, and also automatically applies to UAA too).

Reply:
I agree, but I think it's out of scope for this.
I've created https://trello.com/c/TD9VSPOl/430-fissile-move-validation-code-from-scf-to-fissile

Reply:
Hmm? No, I'm not asking you to move the existing checks; only move any _new_ ones you create (hence would be in scope).